### ISSUE TITLE
Hello, may I ask if you can provide the download address for the dataset?

### ISSUE THREAD

**Issue by WLHao0313** [marco-peer/icdar23#1]

**Comment by marco-peer** [comment]
Hello, may I ask which dataset you are refering to? Full images or patches? If you download ICDAR17/2019, you can generate the patches by using the scripts provided in helpers (see the config files in the config directory for the settings we used for the paper).

**Comment by WLHao0313** [comment]
> 您好，请问您指的是哪个数据集？完整映像还是补丁？如果您下载 ICDAR17/2019，则可以使用帮助程序中提供的脚本生成补丁（有关我们在论文中使用的设置，请参阅配置目录中的配置文件）。
Hello.
1, It would be great if you could provide the patch data, I ran your extract_patch and found that it runs very slowly with my computer configuration (3090+24GB).
[Parallel(n_jobs=2)]: Done   4 tasks      | elapsed:   23.5s
[Parallel(n_jobs=2)]: Done  14 tasks      | elapsed:  1.2min
[Parallel(n_jobs=2)]: Done  28 tasks      | elapsed:  2.2min
2, May I ask how did you train the data?The ICDAR2019 validation dataset is 20GB, which feels big! Did you split the smaller dataset?

Thanks for your answer!

**Comment by marco-peer** [comment]
1. It would be great if you could provide the patch data, I ran your extract_patch and found that it runs very slowly with my computer configuration (3090+24GB).

You can modify the number of cores used (in your snippet, you only use two jobs). There is no GPU support. That should accelerate the process. However, from my experience, the most time consuming step is the clustering, but the script should finish within an hour. 

     2. May I ask how did you train the data?The ICDAR2019 validation dataset is 20GB, which feels big! Did you split the smaller dataset?

The training set for ICDAR2019 consisting of the 32x32 patches is about ~5GB (the full page images are quite large though). The biggest set is the test set (~141 GB). However, you could also do that online (read full image + apply SIFT + inference + aggregation), I will publish this within my other code repository later this month if you are interested in that one. Or do you refer to something else with training? 

I can also provide the dataset to download in case you need it (but I am quite busy right now, probably end of the week).

**Comment by WLHao0313** [comment]
Good afternoon, looking forward to what you do next.

I now have the dataset and have successfully started training. (ps: You used graph neural networks in the program, that's really smart, kudos to you)

For now, I don't have any more questions. I will ask you again when I have problems again.

I hope to keep in touch, good luck.
